definitions/reports/advisor_on_prem_remediations.rb

```diff
@@ -0,0 +1,27 @@
+module Reports
+  class AdvisorOnPremRemediations < ForemanMaintain::Report
+    metadata do
+      description 'Report number of remediations based on advisor rules from advisor on premise'
+      confine do
+        feature(:installer)&.answers&.dig('foreman::plugin::rh_cloud', 'enable_iop_advisor_engine')
+      end
+    end
+
+    def run
+      data_field('advisor_on_prem_remediations') { advisor_on_prem_remediations }
```

Comment:
There is an unwritten convention to end counts with a `_count` suffix.
```suggestion
      data_field('advisor_on_prem_remediation_count') { advisor_on_prem_remediations }
```

Reply:
Applied.